antiplag/tests.py

```diff
@@ -0,0 +1,27 @@
+from django.test import TestCase
+from django.core.mail import EmailMessage
+from django.core.mail import send_mail
+from django.conf import settings
+from .models import Submission
+from .views import SubmissionList
+from django.test import Client
+# Create your tests here.
+
+class EmailTestCase(TestCase):
+    def test_invalid_email(self):
+        """Verify that an invalid email returns correct error message"""
+        c = Client()
+        response = c.post('/api/submissions/', {"email" : "invalid"})
```

Comment:
Please add text to the request. In case that order of checks would rearange, your assertion would fail.